test/basic.jl

```diff
@@ -1,28 +1,29 @@
 using SciMLOperators, LinearAlgebra
 using Random
 
-using SciMLOperators: DiffEqIdentity,
-                      DiffEqNullOperator,
-                      ScaledDiffEqOperator,
-                      AddedDiffEqOperator,
-                      ComposedDiffEqOperator,
-                      DiffEqFunctionOperator
+using SciMLOperators: SciMLIdentity,
+                      SciMLNullOperator,
+                      SciMLScaledOperator,
+                      SciMLAddedOperator,
+                      SciMLComposedOperator,
+
+                      getops
```

Comment:
what is `getops` for?

Reply:
`getops` is used in the `update_coefficients!` interface like

```julia
function update_coefficients!(L::AbstractSciMLOperator, u, p, t)
    for op in getops(L)
        update_coefficients!(op, u, p, t)
    end
    L
end
```

Reply:
was thinking of adding tests for `getops` but didn't get around to do it.

Reply:
oh I see, interesting.